docker/Dockerfile

```diff
@@ -0,0 +1,37 @@
+FROM nvidia/cuda:10.1-cudnn7-devel-ubuntu18.04
+
+ARG PYTHON_VERSION=3.8
+
+# install system-wide package
+RUN apt-get update \
+    && apt-get -y install sudo wget bash openjdk-8-jdk openjdk-8-jre
+
+# set eva user and no password sudo
+RUN useradd -m eva && echo "eva:eva" | chpasswd && adduser eva sudo
+RUN echo "eva ALL=(ALL) NOPASSWD: ALL" >> /etc/sudoers
+
+USER eva
+
+# install miniconda
+RUN wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh -O ~/miniconda.sh \
+    && chmod +x ~/miniconda.sh \
+    && bash ~/miniconda.sh -b -p ~/miniconda \
+    && hash -r \
+    && ~/miniconda/bin/conda config --set always_yes yes --set changeps1 no \
+    && ~/miniconda/bin/conda update -q conda \
+    && ~/miniconda/bin/conda info -a
+
+COPY . /app
+RUN sudo chown -R eva:eva /app
+
+# install eva env
+RUN ~/miniconda/bin/conda env create -f /app/script/install/conda_eva_environment.yml
+
+# sanity check for GPU
+RUN ~/miniconda/bin/conda run -n eva python -c "import torch; torch.randn(1).cuda()"
```

Comment:
```
ERROR conda.cli.main_run:execute(31): Subprocess for 'conda run ['python', '-c', 'import torch; torch.randn(1).cuda()']' command failed.  (See above for error)
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/home/eva/miniconda/envs/eva/lib/python3.7/site-packages/torch/cuda/_init_.py", line 192, in _lazy_init
    _check_driver()
  File "/home/eva/miniconda/envs/eva/lib/python3.7/site-packages/torch/cuda/_init_.py", line 102, in _check_driver
    http://www.nvidia.com/Download/index.aspx""")
AssertionError: 
Found no NVIDIA driver on your system. Please check that you
have an NVIDIA GPU and installed a driver from
http://www.nvidia.com/Download/index.aspx
```
Encountered above error when tested on ada1

Reply:
Add a CPU only test case to bypass the GPU dependency. 
https://github.com/georgia-tech-db/eva/pull/103/files#diff-4e5e90c6228fd48698d074241c2ba760R32